examples/wasm/envoy_filter_http_wasm_example.cc

```diff
@@ -32,19 +243,122 @@ bool ExampleRootContext::onStart(size_t) {
   return true;
 }
 
-void ExampleContext::onCreate() { LOG_WARN(std::string("onCreate " + std::to_string(id()))); }
+bool ExampleRootContext::onConfigure(size_t config_size) {
```

Comment:
will it have problems if this function is called multiple times and the same config is reused? or is this function only called once?

Reply:
onConfigure is called only once when the module starts running. onCreate is called upon every new request.